spectrum-offchain/src/backlog.rs

```diff
@@ -342,8 +354,30 @@ where
 
     async fn remove<'a>(&mut self, ord_id: TOrd::TOrderId)
     where
-        TOrd::TOrderId: 'a,
+        TOrd::TOrderId: Clone + 'a,
     {
+        let w = self
+            .pending_pq
+            .iter()
+            .find(|o| o.0.order_id == ord_id)
+            .map(|w| w.0.clone());
+        if let Some(w) = w {
+            self.pending_pq.remove(&w);
+        }
+
+        let w = self
+            .suspended_pq
+            .iter()
+            .find(|o| o.0.order_id == ord_id)
+            .map(|w| w.0.clone());
+        if let Some(w) = w {
+            self.suspended_pq.remove(&w);
+        }
+
+        if let Some(ix) = self.revisit_queue.iter().position(|o| o.order_id == ord_id) {
+            self.revisit_queue.remove(ix);
+        }
```

Comment:
we don't need to clean queues actively. Instead, we already do it passively - in `try_pop_max_order` when `order_id` is not found in persistent store we just move to the next order.
https://github.com/spectrum-finance/spectrum-offchain-ergo/commit/46c3c26bea3ea9236422520b818ebb1131d795cb

Reply:
Ah yes, of course. Removed.